pulsar-client-api/src/main/java/org/apache/pulsar/client/api/ConsumerBuilder.java

```diff
@@ -200,7 +200,7 @@
     ConsumerBuilder<T> ackTimeout(long ackTimeout, TimeUnit timeUnit);
 
     /**
-     * Ack returns receipt but does not mean that the message is not resent after get receipt.
+     * Ack returns receipt, but the message is not re-sent after getting receipt.
```

Comment:
I would use `Acknowledgement`, not `Ack` in docs.

Reply:
Also I think the should be clarified. As I understand, this is used for transactions. It should be explained what ack receipts are for.